design-documents/storefront/Readme.md

```diff
@@ -0,0 +1,15 @@
+# Storefront
```

Comment:
Should put design documents related to storefront application in the storefront repo? We already have some design documents there. If not, it would make sense to move all design documents to this repo.

Reply:
I thought this repo is source for all design docs. I added a link to this PR in the SF wiki sidebar, so people can find it. When the document is approved and merged, I'll update the link.